x-pack/platform/plugins/shared/ml/public/application/explorer/explorer_constants.ts

```diff
@@ -22,7 +22,6 @@ export const EXPLORER_ACTION = {
   CLEAR_INFLUENCER_FILTER_SETTINGS: 'clearInfluencerFilterSettings',
   CLEAR_JOBS: 'clearJobs',
   JOB_SELECTION_CHANGE: 'jobSelectionChange',
-  SET_CHARTS_DATA_LOADING: 'setChartsDataLoading',
```

Comment:
- It looks like in this file `DRAG_SELECT_ACTION` is no longer used and can be removed (probably a leftover of the previous swim lane version).
- As far as I can see the whole of `EXPLORER_ACTION` is now also no longer used.

Reply:
Good find! It's hard to catch all the unused leftovers, done in: [#e2e5c00](https://github.com/elastic/kibana/pull/203224/commits/e2e5c006dff8c443062996b3b069b03913f75ec0)